src/Layout/Presentation/RemainderQuotients/RemainderQuotients.tsx

```diff
@@ -74,15 +74,8 @@ export class RemainderQuotients extends React.Component<RemainderQuotientsProps>
                         return (
                             <div
                                 className={row.value.wonLevellingSeat ? "has-background-dark has-text-white" : ""}
-                                style={
-                                    {
-                                        /* textAlign: "center",
-                                    color: row.value.wonLevellingSeat ? "white" : "black",
-                                    backgroundColor: row.value.wonLevellingSeat ? "#ff6e00" : "white", */
-                                    }
-                                }
                             >
-                                {Number(quotient).toFixed(this.props.decimals)}
+                                {replaceComma(Number(quotient).toFixed(this.props.decimals))}
```

Comment:
How come replaceComma is used instead of the numberFormatFraction function?

Reply:
This comment is attached to the conversation in customNumberFormat.

Reply:
I did not want to separate the thousands, a design choice. Ofc if we agree on separating the thousands, for remainder quotients, we do that also.

Reply:
Why would we be inconsistent in how we format numbers?

Reply:
It looked a bit weird when the thousands were separated in a number with decimals. It also took more space. I guess I'm wrong. It is a common style. let's go for consistency!